scss/tech.scss

```diff
@@ -54,7 +53,7 @@
     }
 
     @media screen and (max-width: 900px) {
```

Comment:
Do media queries out of this css class, it should be general for all classes. You should need only one, we usually use 1000px as breakpoint.